src/Components/Server/src/Circuits/CircuitJSComponentInterop.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json;
+using Microsoft.AspNetCore.Components.Web;
+using Microsoft.AspNetCore.Components.Web.Infrastructure;
+using Microsoft.JSInterop;
+
+namespace Microsoft.AspNetCore.Components.Server.Circuits
+{
+    /// <summary>
+    /// Intended for framework use only. Not supported for use from application code.
+    /// </summary>
+    public class CircuitJSComponentInterop : JSComponentInterop
```

Comment:
Does `[EditorBrowsable(EditorBrowsableState.Never)]` gets inherited?
https://github.com/dotnet/aspnetcore/blob/ae571c13ffff1e6d2c1cd01c3ebbfd1524221792/src/Components/Web/src/JSComponents/JSComponentInterop.cs#L22-L23

```suggestion
    [EditorBrowsable(EditorBrowsableState.Never)]
    public class CircuitJSComponentInterop : JSComponentInterop
```